packages/blockchain-wallet-v4-frontend/src/data/components/brokerage/sagas.ts

```diff
@@ -263,6 +263,24 @@ export default ({ api, coreSagas, networks }: { api: APIType; coreSagas: any; ne
     }
   }
 
+  const handleWithdrawClick = function* ({ payload }: ReturnType<typeof A.handleWithdrawClick>) {
+    yield put(actions.components.withdraw.showModal(payload.fiatCurrency))
+
+    const bankTransferAccountsR = selectors.components.brokerage.getBankTransferAccounts(
+      yield select()
+    )
+    const bankTransferAccounts = bankTransferAccountsR.getOrElse([])
+    if (bankTransferAccounts.length) {
+      yield put(
+        actions.components.brokerage.setBankDetails({
+          account: bankTransferAccounts.filter(
+            (a) => a.currency === payload.fiatCurrency && a.state === 'ACTIVE'
```

Comment:
maybe move 'ACTIVE' to be enum

Reply:
I'm not exactly sure how it works. I've just changed it for the linter.